Src/Newtonsoft.Json/Utilities/ConvertUtils.cs

```diff
@@ -930,6 +930,137 @@ public static ParseResult Int64TryParse(char[] chars, int start, int length, out
+                char c = chars[i];
+                switch (chars[i])
+                {
+                    case '.':
+                        numDecimalStart = i + 1;
+                        break;
+                    case 'e':
+                    case 'E':
+                        {
+                            i++;
+                            if (i == end)
+                                return ParseResult.Invalid;
+
+                            if (numDecimalStart < end)
+                                numDecimalEnd = i - 1;
+
+                            c = chars[i];
+                            switch (c)
+                            {
+                                case '-':
+                                    exponentNegative = true;
+                                    i++;
+                                    break;
+                                case '+':
+                                    i++;
+                                    break;
+                            }
+
+                            // parse 3 digit 
+                            for (;i < end;i++)
+                            {
+                                c = chars[i];
+                                if (c < '0' || c > '9')
+                                    return ParseResult.Invalid;
+
+                                exponent = (10 * exponent) + (c - '0');
+                            }
+
+                            if (exponentNegative)
+                            {
+                                exponent = -exponent;
+                            }
+                            else
+                            {
+                                if (exponent > MaxExponent)
+                                {
+                                    value = isNegative ? double.NegativeInfinity : double.PositiveInfinity;
+                                    return ParseResult.Success;
+                                }
+                            }
+                        }
+                        break;
+                    case '0':
+                    case '1':
+                    case '2':
+                    case '3':
+                    case '4':
+                    case '5':
+                    case '6':
+                    case '7':
+                    case '8':
+                    case '9':
+                        value = (10 * value) + (c - '0');
+                        break;
+                    default:
+                        return ParseResult.Invalid;
+                }
+            }
+
+            // correct the decimal point
+            exponent -= (numDecimalEnd - numDecimalStart);
+
+            if (exponent > 0)
+            {
+                value *= DoubleExponents[exponent - 1];
+            }
+            else if (exponent < 0)
+            {
+                exponent = -exponent;
+                if (exponent > MaxExponent)
+                {
+                    // cover 4.94065645841247E-324
```

Comment:
Dumb question but is this to cover all large negative exponents or just this one particular one?